appgate_sdp/assets/monitors/monitor_Active_connection_exceeds_limit.json

```diff
@@ -0,0 +1,36 @@
+{
+	"version": 2,
+	"created_at": "2025-04-02",
+	"last_updated_at": "2025-04-02",
+    "title": "Active connection exceeds limit",
+    "description": "The appliance has a high amount of connections",
+	"definition": {
+		"id": 168100563,
+		"name": "Active connection exceeds limit",
+		"type": "query alert",
+		"query": "avg(last_5m):(sum:appgate.apn_active_connections{*} by {collective_name,site_name,appliance_name,func} / sum:appgate.apn_active_connections_max{*} by {collective_name,site_name,appliance_name,func}) * 100 > 90",
+		"message": "Appliance: {{appliance_name.name}} \nThe appliance has a high amount of connections",
+		"tags": [
+			"func:apn",
+			"service:appgate"
+		],
+		"options": {
+			"thresholds": {
+				"critical": 90,
+				"warning": 80
+			},
+			"notify_audit": false,
+			"include_tags": true,
+			"new_group_delay": 60,
+			"silenced": {},
+			"on_missing_data": "default"
+		},
+		"priority": null,
+		"restriction_policy": {
+			"bindings": []
+		}
+	},
+	"tags": [
+		"integration:appgate-sdp"
```

Comment:
```suggestion
		"integration:appgate_sdp"
```

I think?

Reply:
I thought the tag here had to be the `app_id` in the manifest: https://github.com/DataDog/integrations-core/blob/master/appgate_sdp/manifest.json#L4.

`aws_neuron` uses it correctly: https://github.com/DataDog/integrations-core/blob/master/aws_neuron/assets/monitors/execution_errors.json#L26-L28.

Reply:
But I do think we also add a secondary integration with snake case! I'll add it in